go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/hashicorp/terraform-plugin-docs v0.13.0
-	github.com/hashicorp/terraform-plugin-framework v0.11.1
+	github.com/hashicorp/terraform-plugin-framework v0.11.2-0.20220909031638-99f9980b5305
```

Comment:
As you mentioned in the description, just flagging that this will need updating to `v0.11.2` once https://github.com/hashicorp/terraform-plugin-framework/pull/472 is merged.